### PR TITLE
Feat: #1 Sharded CAS API annotation calls

### PR DESCRIPTION
**Pull request by fedorgrab**

### CODE REVIEW

File: src/casp_cli/service.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                )
+            )
+            i = j
+            j += chunk_size
+
+        await asyncio.wait(tasks)
+
+    def annotate_anndata(self, adata: "anndata.AnnData", chunk_size=2000) -> t.List:
+        start = time.time()
+        self._print("CAS v1.0 (Model ID: PCA_002)")
```
Comment (kcibul) [inline src/casp_cli/service.py:97]: call a function to get these, that could be static for now but really should come from the API somehow (or be parameters from the user as to which model they want to use).

File: test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+aiohttp[speedups]
```
Comment (kcibul) [inline test-requirements.txt:1]: same question about dev-requirements

**Review by kcibul — APPROVED** [review]
Looks good as a v1 -- a few minor comments throughout.  We talked about this at standup, but some basic retry logic is important to add.  Even for things like timeouts, 503s, etc having an exponential backoff with jitter and a fixed number of retries on non-200 attempts would be a great start.

File: dev-requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+aiohttp[speedups]
```
[the remark this answers is not in the document]
Comment (fedorgrab) [reply dev-requirements.txt:1]: Honestly had no idea how it works.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,25 @@
-# cell-annotation-service-client
+# Cell Annotation Service Client
```
[the remark this answers is not in the document]
Comment (fedorgrab) [reply README.md:1]: Fully agree, will do it.

File: .github/workflows/tox.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+          python -m pip install --upgrade pip
+          pip install -r dev-requirements.txt
+      - name: Install Tox
+        run: pip install tox
+
+      - name: Run Tox lint commands
+        # Run tox using the version of Python in `PATH`
+        run: tox -e lint
+
+        # TODO: Uncomment when have at least 1 unit test
```
[the remark this answers is not in the document]
Comment (fedorgrab) [reply .github/workflows/tox.yaml:39]: Agreed.

**Pull request merged**